test/dlc_tests/container_tests/bin/benchmark/get_avg_accuracies.py

```diff
@@ -0,0 +1,27 @@
+# usage: python <script_name>.py <log_file> 4 Training & Inference GPU
+# usage: python <script_name>.py <log_file> 1 Inference CPU
+import sys
+
+
+def get_avg_speed(filepath, index):
+    total = 0.0
+    n = 0
+    with open(filepath) as f:
+        for line in f:
+            if "Speed" in line:
```

Comment:
It this terminology (i.e., searching  for "Speed" specifically) valid for across all frameworks, or is it specific to MXNet?

Reply:
This is a MXNet specific script. It will only be called by MXNet benchmark functions. Should I modify the file name to "get_MXNet_avg_accuracies.py" or some other description?

Reply:
Yes, "get_mxnet_benchmark_avg_accuracies.py" or something similar would be better.